src/controllers/authSignup.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-len */
+/* eslint-disable no-shadow */
+const bcrypt = require('bcrypt');
+const { userSignupQuery } = require('../database/queries');
+const { userSignupSchema } = require('../utils/validation');
+const { generateAccessToken } = require('../utils/jwt');
+
+const saltRounds = 10;
```

Comment:
Consider storing the salt value in the environment file for improved security and flexibility.
Access the salt value using `process.env.SALT_ROUNDS` to separate configuration details.
This ensures better maintainability and keeps sensitive information secure.